CONTRIBUTING.md

```diff
@@ -0,0 +1,130 @@
+Introduction
+============
+
+This document summarizes the general guidelines for contributing to this project.
+
+Basic Workflow
+==============
+
+Every developer should follow the following workflow when contributing to the
+project:
+
+Initial setup
+-------------
+
+To setup your environment, follow these steps:
+
+1. Go to the [`CarND-Eurobots/CarND-Capstone`](https://github.com/CarND-EuroBots/CarND-Capstone) repository and click on `Fork`. This will create a fork of the repo in
```

Comment:
Is there an option to open pull requests without forking the project? Could we just create our feature branches in the main repository and open pull requests there?

Reply:
Sure! The Fork option is what I've seen in many projects but I guess in our case it's fine to just create branches in the main repository. I also find it quite cumbersome to have a Fork especially when updating from upstream.

I'll change that!

Reply:
Done :)

Reply:
Great!